### PR TITLE
Added reference to IANA language subtag registry

### PR DESCRIPTION
**Pull request by william-vw**

### CODE REVIEW

File: refs/biblio.json [hunk 1]
```diff
@@ -1935,6 +1935,11 @@
         "status": "W3C Incubator Group Report",
         "publisher": "W3C"
     },
+	"LNG-TAG": {
+		"title": "Language Subtag Registry",
+		"href": "http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry",
+		"publisher": "IANA"
+	},
```
Comment (ExE-Boss) [inline refs/biblio.json:1942]:
```suggestion
    "LNG-TAG": {
        "title": "Language Subtag Registry",
        "href": "http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry",
        "publisher": "IANA"
    },
```

**Comment by william-vw** [comment]
(note that all tests were passed ("npm test") - so unclear why the spaces are an issue)

File: refs/biblio.json [hunk 2]
```diff
@@ -1935,6 +1935,11 @@
         "status": "W3C Incubator Group Report",
         "publisher": "W3C"
     },
+    "LNG-TAG": {
```
Comment (marcoscaceres) [inline refs/biblio.json:1938]: Maybe more readable?: 

```suggestion
    "LANG-SUBTAG-REGISTRY": {
```

Comment (marcoscaceres) [reply refs/biblio.json:1938]: Also less change it will be used to cite BCP47.

**Comment by tobie** [comment]
> This repository uses spaces.

That'll get cleaned-up by the bot. No need to worry.

**Pull request merged**